src/main/java/cloud/fogbow/ras/core/plugins/interoperability/googlecloud/sdk/v1/network/models/InsertNetworkResponse.java

```diff
@@ -3,27 +3,28 @@
 import cloud.fogbow.common.util.GsonHolder;
 import com.google.gson.annotations.SerializedName;
 import static cloud.fogbow.ras.core.plugins.interoperability.googlecloud.util.GoogleCloudConstants.Network.NETWORK_KEY_JSON;
+import static cloud.fogbow.ras.core.plugins.interoperability.googlecloud.util.GoogleCloudConstants.Network.TARGET_LINK_KEY_JSON;
```

Comment:
Better use import, instead of import static, since this will force you to explicitly refer to the static class name in the code, making it clearer that you are invoking a method from a static class. This leaves the code more readable and maintainable.